README.md

```diff
@@ -52,21 +52,21 @@ These configurations are officially supported.
 
 | Configuration | Supported OS / hardware |
 | :------------ | :---------------------- |
-| generic_x86-64_config | Generic x86, 64bit distribution *use this as a starting point only!*  |
-| raspberrypi/uboot/raspberrypi0w_config | RaspberryPi 0w, Raspbian 32bit |
-| raspberrypi/uboot/debian/raspberrypi3_bookworm_64bit_config | RaspberryPi 3, Raspberry Pi OS "bookworm" 64bit |
-| raspberrypi/uboot/debian/raspberrypi4_bookworm_64bit_config | RaspberryPi 4, Raspberry Pi OS "bookworm" 64bit |
-| raspberrypi/uboot/ubuntu/raspberrypi4_ubuntu_32bit_config | RaspberryPi 4, Ubuntu 32bit |
+| configs/generic_x86-64_hdd_config | Generic x86, 64bit distribution *use this as a starting point only!*  |
```

Comment:
This one is a good catch (thanks!) but for the reset I don't think we need the `configs/` prefix - that part is quite obvious.

Why I wrote in the JIRA task aboud adjusting the README was for the commands that used these files. You haven't seen it because [I see now that] we fixed them in master (in #727) but didn't port to 4.3.x (and then both you and me use the wrong configs during release testing) -> Anyway, this part I took care of in #731